content/references/rippled-api/public-rippled-methods/transaction-methods/tx.md

```diff
@@ -197,11 +201,79 @@ The response follows the [standard format][], with a successful result containin
 | `validated`    | Boolean          | True if this data is from a validated ledger version; if omitted or set to false, this data is not final. |
 | (Various)      | (Various)        | Other fields from the [Transaction object](transaction-formats.html) |
 
+### Not Found Response
+
+If the server does not find the transaction, it returns a `txnNotFound` error, which could mean two things:
+
+- The transaction has not been included in any ledger version, and has not been executed.
+- The transaction was included in a ledger version that the server does not have available.
+
+This means that a `txnNotFound` on its own is not sufficient to know the [final outcome of a transaction](finality-of-results.html).
+
+To further narrow down the possibilities, you can provide a range of ledgers to search using the `min_ledger` and `max_ledger` fields in the request. If you provide **both** of those fields, the `txnNotFound` response includes the following field:
+
+| `Field`        | Type      | Description                              |
+|:---------------|:----------|:-----------------------------------------|
+| `searched_all` | Boolean   | _(Omitted unless the request provided `min_ledger` and `max_ledger`)_ If `true`, the server was able to search all of the specified ledger versions, and the transaction was in none of them. If `false`, the server did not have all of the specified ledger versions available, so it is not sure if one of them might contain the transaction. [New in: rippled 1.5.0][] |
+
+An example of a `txnNotFound` response that fully searched a requested range of ledgers:
+
+<!-- MULTICODE_BLOCK_START -->
+
+_WebSocket_
+
+```json
+{
+  "error": "txnNotFound",
+  "error_code": 29,
+  "error_message": "Transaction not found.",
+  "id": 1,
+  "request": {
+    "binary": false,
+    "command": "tx",
+    "id": 1,
+    "max_ledger": 54368673,
+    "min_ledger": 54368573,
+    "transaction": "E08D6E9754025BA2534A78707605E0601F03ACE063687A0CA1BDDACFCD1698C7"
+  },
+  "searched_all": true,
+  "status": "error",
+  "type": "response"
+}
+```
+
+_JSON-RPC_
+
+```json
+200 OK
+
+{
+  "result": {
+    "error": "txnNotFound",
+    "error_code": 29,
+    "error_message": "Transaction not found.",
+    "request": {
+      "binary": false,
+      "command": "tx",
+      "max_ledger": 54368673,
+      "min_ledger": 54368573,
+      "transaction": "E08D6E9754025BA2534A78707605E0601F03ACE063687A0CA1BDDACFCD1698C7"
+    },
+    "searched_all": true,
+    "status": "error"
+  }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
 ## Possible Errors
 
 * Any of the [universal error types][].
 * `invalidParams` - One or more fields are specified incorrectly, or one or more required fields are missing.
 * `txnNotFound` - Either the transaction does not exist, or it was part of an older ledger version that `rippled` does not have available.
+* `excessiveLgrRange` - The `min_ledger` and `max_ledger` fields of the request were more than 1000 apart.
```

Comment:
"were more" - "are more" seems to be more common to use, although I understand the time sequencing issue here.